config.js

```diff
@@ -143,7 +144,7 @@ module.exports = {
   // Only for when nostr client requesting server information.
   server_meta: {
     "contact": "unset",
-    "pubkey": "0000000000000000000000000000000000000000000000000000000000000000",
+    "pubkey": "npub1m78s5eqv8l7snc5nnxdvlgue6pt5epgplndtem99quhwyptas7jss2qx53",
```

Comment:
Convert the npub into hex string.
You could do that by running the following:
```
bostr hexconverter npub1.....
```

Or:
```
node hexconverter.js npub1.....
```